src/components/library-camera/screens/show-book.js

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect} from 'react';
+import {StyleSheet, Text, View, Image} from 'react-native';
+
+import {formConfig} from '../../../constants/book-form';
+import {startCase} from '../../../helpers/global';
+
+const {photo} = formConfig;
+
+const {emptyArr} = global;
+
+const ShowBook = ({navigation, route: {params}}) => {
+  useEffect(() => {
+    navigation.setOptions({
+      title: `${params.bookName} Details`,
+    });
+  }, emptyArr);
+
+  const {container, label, bookInfo, image} = styles;
+
+  return (
+    <View style={container}>
+      {params &&
+        Object.entries(params).map(([key, value]) => (
+          <View key={key} style={bookInfo}>
+            <Text style={label}>{startCase(key)}</Text>
+            {key == photo.key ? (
```

Comment:
Image should always be shown (fallback to default image) why are we putting key condition  ?